### PR TITLE
feat(news-app)fetches api with a function

### LINKED ISSUE #1: Fetch/Set up Api Function
**Is your feature request related to a problem? Please describe.
Make API fetch to grab data for app

**Describe the solution you'd like
Make functions to fetch API

### PR DESCRIPTION
**Pull request by JorgeLVilla**
## Changes
_List Changes Introduced by this PR_
1.created a function that fetches the API information.

## Purpose
To get the information we need from the API.

Closes #1

### CODE REVIEW

File: modern-news-app/src/App.tsx [hunk 1]
```diff
@@ -1,24 +1,13 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { fetchNewsData } from './util/fetchApi';
 
 function App() {
+  console.log(fetchNewsData("bitcoin"))
```
Comment (connielion) [inline modern-news-app/src/App.tsx:7]: Do you need this still?

File: modern-news-app/src/util/fetchApi.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+// PageNumber for Api endpoint
+const pageNumber: string = "&page=1"
+// sortBy param for api: relevancy, popularity, publishedAt
+const sortBy: string = "&sortBy=popularity";
+
+
+// return news data with a "q" search parameter
+export const fetchNewsData = async (searchQuery: string) => {
+    try {
+        const baseUrl = `https://newsapi.org/v2/everything?q=${searchQuery}&apiKey=${process.env.REACT_APP_NEWS_API_KEY}${pageSize}${pageNumber}${sortBy}`;
```
Comment (krferrell) [inline modern-news-app/src/util/fetchApi.ts:12]: Looking good. Nice work.

**Review by krferrell — APPROVED** [review]
Nice job type checking on the different parameters

**Review by Gjhnsn — APPROVED** [review]
Looking good!

File: modern-news-app/src/App.tsx (hunk 1 above)
Comment (JorgeLVilla) [reply modern-news-app/src/App.tsx]: No, we don't. Thanks for letting me know!

**Pull request merged**